From 4133863cd6d8fbda3ff3e2131813848f2114fd32 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Thu, 3 Dec 2009 14:20:11 -0500
Subject: [PATCH 36/43] vbi: int_pend --> irq_pend

Names using "int" are misleading due to the classic variable type
(ie, int, long, char) -- so rather use irq to be consistent with
the interrupt context and with the other vars already like this.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 arch/powerpc/kernel/vbi/wrhv.c |    6 +++---
 include/vbi/compat.h           |    2 +-
 include/vbi/interface.h        |    6 +++---
 include/vbi/vbi.h              |    6 +++---
 include/vbi/vdk_interface.h    |    2 +-
 kernel/vbi/show.c              |    2 +-
 6 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/powerpc/kernel/vbi/wrhv.c b/arch/powerpc/kernel/vbi/wrhv.c
index b0e0ed8..fbe5e44 100644
--- a/arch/powerpc/kernel/vbi/wrhv.c
+++ b/arch/powerpc/kernel/vbi/wrhv.c
@@ -307,17 +307,17 @@ unsigned int wrhv_vioapic_get_irq(void)
 {
 	unsigned int irq;
 
-	irq = wr_control->int_pend;
+	irq = wr_control->irq_pend;
 
 #ifdef CONFIG_DEBUG_VIRTUAL_IRQS
 	/* Maybe this is useless for real external interrupt */
-	wr_status->int_pend = 0;
+	wr_status->irq_pend = 0;
 #endif
 
 	if (irq == 0xffff)
 		irq = NO_IRQ_IGNORE;
 	else
-		wr_control->int_pend = 0xffff;
+		wr_control->irq_pend = 0xffff;
 
 	return irq;
 }
diff --git a/include/vbi/compat.h b/include/vbi/compat.h
index 2cf7bad..c909d0c 100644
--- a/include/vbi/compat.h
+++ b/include/vbi/compat.h
@@ -84,7 +84,7 @@
 #define numSm			num_sm
 #define numMem			num_mem
 
-#define intPending		int_pend
+#define intPending		irq_pend
 #define resetPC			reset_pc
 #define bootCount		boot_count
 #define boardName		board_name
diff --git a/include/vbi/interface.h b/include/vbi/interface.h
index 02f4a96..0e62485 100644
--- a/include/vbi/interface.h
+++ b/include/vbi/interface.h
@@ -318,7 +318,7 @@ struct vb_control
 					/* Virtual I/O APIC */
 	VB_ALIGN_FIELD_64 (void *vIoapic, pad3);
 
-	uint32_t int_pend;		/* actual virtual interrupt pending */
+	uint32_t irq_pend;		/* actual virtual interrupt pending */
 
 };
 
@@ -365,7 +365,7 @@ struct vb_control
  *       |		|<-------64bits - VB_ARCH_STATUS_REGS; arch dependent
  *       |		|		structure pointer
  *       |--------------|
- *       |		|<-------32bits - int_pend; pending vector number
+ *       |		|<-------32bits - irq_pend; pending vector number
  *       |--------------|
  *       |//////////////|<-------32bits - Reserved for future enhencements
  *       |--------------|
@@ -395,7 +395,7 @@ struct vb_status {
 #endif
 
 	/* Bits indicating which interrupts are pending */
-	uint32_t int_pend_type;
+	uint32_t irq_pend_type;
 
 	/* reserved field for future use, required for timeStamp alignment */
 	uint32_t reserved1;
diff --git a/include/vbi/vbi.h b/include/vbi/vbi.h
index 96c1402..eeeaffc 100644
--- a/include/vbi/vbi.h
+++ b/include/vbi/vbi.h
@@ -53,7 +53,7 @@ various data structures.
 
 
 #define VBI_INT_PENDING_SET(value)        \
-        (VBI_CNTRL_ADDR_GET()->int_pend = value)
+        (VBI_CNTRL_ADDR_GET()->irq_pend = value)
 
 /*
  *
@@ -137,7 +137,7 @@ various data structures.
  *
  */
 #define VBI_INT_VCORE_PENDING()				    \
-        (VBI_CNTRL_ADDR_GET()->int_pend)
+        (VBI_CNTRL_ADDR_GET()->irq_pend)
 
 /*
  * VBI_INT_VCORE_PENDING_TYPE_GET - Get the pending interrupts
@@ -149,7 +149,7 @@ various data structures.
  *	    0 if none is pending
  */
 #define VBI_INT_VCORE_PENDING_TYPE_GET()            \
-        (VBI_STATUS_ADDR_GET()->int_pend_type)
+        (VBI_STATUS_ADDR_GET()->irq_pend_type)
 
 /*
  * VBI_INT_VCORE_PREVSTATE_GET - Get the core's previous interrupt status 
diff --git a/include/vbi/vdk_interface.h b/include/vbi/vdk_interface.h
index 8c165e7..bf2af37 100644
--- a/include/vbi/vdk_interface.h
+++ b/include/vbi/vdk_interface.h
@@ -318,7 +318,7 @@ typedef struct td_status
 /* Do not change the order of these fields */
 struct vb_status
 {
-    unsigned int int_pend_type; /* 0: Bits indicating which interrupts are pending */
+    unsigned int irq_pend_type; /* 0: Bits indicating which interrupts are pending */
 
     /* reserved field for future use, required for timeStamp alignment */
     unsigned long reserved1; /* 1: */
diff --git a/kernel/vbi/show.c b/kernel/vbi/show.c
index 5293064..a1e656a 100644
--- a/kernel/vbi/show.c
+++ b/kernel/vbi/show.c
@@ -118,7 +118,7 @@ void vbi_show_stat(void)
 	struct vb_status *p = VBI_STATUS_ADDR_GET();
 	printk("VB status: 0x%p\n", p);
 
-	printk("  Pending interrupts: 0x%08x\n", p->int_pend_type);
+	printk("  Pending interrupts: 0x%08x\n", p->irq_pend_type);
 	printk("  timestamp:          %lld\n", p->timeStamp);
 	printk("  old int disable:    0x%08x\n", p->prev_irq_disable);
 #ifdef _WRHV_ARCH_HAS_STATUS_REGS
-- 
1.6.5.2

